go/libkb/constants.go

```diff
@@ -576,10 +576,11 @@ const (
 const (
 	ServiceLogFileName = "keybase.service.log"
 	EKLogFileName      = "keybase.ek.log"
+	GUILogFileName     = "keybase.gui.log"
 	KBFSLogFileName    = kbconst.KBFSLogFileName
 	GitLogFileName     = "keybase.git.log"
 	UpdaterLogFileName = "keybase.updater.log"
-	DesktopLogFileName = "Keybase.app.log"
+	DesktopLogFileName = "Keybase.app.log" // deprecated; use GUILogFileName
```

Comment:
why not use keep the same filename?

Reply:
I was afraid I'd mess it up. But I guess it's not a huge deal to lose some logs here .. will update to use the same name.

Reply:
actually I do want to lowercase that. So to avoid confusing maybe it's best to keep the `.gui` part?

Reply:
ended up keeping `Keybase.app.log`